BE/OwnershipAndControl/CorporateControl.rdf

```diff
@@ -264,55 +269,62 @@
-		<rdfs:domain rdf:resource="&fibo-be-le-cb;StockCorporation"/>
+		<rdfs:domain rdf:resource="&fibo-be-le-lp;LegalEntity"/>
 		<rdfs:range rdf:resource="&fibo-be-oac-cctl;Subsidiary"/>
-		<skos:definition>relates a company to one of its subsidiaries, that is an affiliae controlled by the company directly, or indirectly through one or more intermediaries</skos:definition>
+		<skos:definition>relates a legal entity to another organization that it controls directly, or indirectly, through one or more intermediaries</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-be-oac-cctl;holdsMajorityControllingVotingRightsIn">
 		<rdfs:subPropertyOf rdf:resource="&fibo-be-oac-cctl;holdsSignificantControllingVotingRightsIn"/>
 		<rdfs:label>holds majority controlling voting rights in</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-be-le-lp;LegalEntity"/>
-		<skos:definition>relates a legal person to a company in which the party holds fifty percent or more of the controlling voting rights</skos:definition>
+		<skos:definition>relates a party to a legal entity in which it holds fifty percent or more of the controlling voting rights</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-be-oac-cctl;holdsSignificantControllingVotingRightsIn">
 		<rdfs:subPropertyOf rdf:resource="&fibo-be-oac-cctl;holdsSomeControllingVotingRightsIn"/>
 		<rdfs:label>holds significant controlling voting rights in</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-be-le-lp;LegalEntity"/>
-		<skos:definition>relates a legal person to a company in which the party holds a significant proportion of the controlling voting rights</skos:definition>
+		<skos:definition>relates a party to a legal entity in which it holds a significant proportion of the controlling voting rights</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-be-oac-cctl;holdsSomeControllingVotingRightsIn">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;controls"/>
 		<rdfs:label>holds some controlling voting rights in</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-oac-ctl;ControllingParty"/>
 		<rdfs:range rdf:resource="&fibo-be-le-lp;LegalEntity"/>
-		<skos:definition>relates a legal person to a company in which that party holds some controlling voting rights</skos:definition>
+		<skos:definition>relates a party to a legal entity in which it holds some controlling voting rights</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-be-oac-cctl;isAffiliateOf">
+		<rdf:type rdf:resource="&owl;SymmetricProperty"/>
+		<rdfs:label>is affiliate of</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-be-oac-cctl;Affiliate"/>
+		<rdfs:range rdf:resource="&fibo-be-oac-cctl;Affiliate"/>
+		<skos:definition>relates a party which directly, or indirectly through one or more intermediaries, controls, or is controlled by, or is under common control another party</skos:definition>
```

Comment:
missing word - should be "of another party" at the end.
This definition definitely needs a source.
I think it would exclude a joint venture and one of its (50%) joint owners.

Reply:
fixed